docs/reference/functions.md

```diff
@@ -0,0 +1,157 @@
+# Core Functions
+
+Rules use "functions" and those can be custom defined. To save everyone the effort of writing functions for common tasks, Spectral comes with a few bundled out of the box. 
+
+All functions 
```

Comment:
Hey, is it an oversight? Maybe missing a comma or should be a header?

Reply:
Oversight! Great catch.